### PR TITLE
Added tlg0542.tlg001.1st1K-grc1.xml

### PR DESCRIPTION
**Pull request by sonofmun**
Small portion of Julius Pollux Onomasticon

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +3 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |     +130 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0542/__cts__.xml`                          | New      |
| `./data/tlg0542/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0542/__cts__.xml`                          | Passing  |
| `./data/tlg0542/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3721)*

File: data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	 <name>Frederick Hultsch</name>
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0542">Julius Pollux</author>
+  <imprint>
+	<publisher>Teubner</publisher>
+	<pubPlace>Leipzig</pubPlace>
+	<date>1864</date>
+  </imprint>
+  <biblScope unit="volume">I</biblScope>
```
Comment (planatheisa) [inline data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml]: Should not be a roman numeral.

File: data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+ </monogr>
+ </biblStruct>
+</listBibl>
+</sourceDesc>
+</fileDesc>
+<encodingDesc>
+ <p>Text encoded in accordance with the latest EpiDoc standards</p>
+ <refsDecl n="CTS">
```
Comment (planatheisa) [inline data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml]: The CTS mentions `book` and `chapter`, the text has `book` and `section`. And I think one level should be sufficient.

File: data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+	</persName>
+  </editor>
+  <author ref="urn:cts:greekLit:tlg0542">Julius Pollux</author>
+  <imprint>
+	<publisher>Teubner</publisher>
+	<pubPlace>Leipzig</pubPlace>
+	<date>1864</date>
+  </imprint>
+  <biblScope unit="volume">I</biblScope>
+ </monogr>
```
Comment (planatheisa) [inline data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml]: No scan.

**Comment by sonofmun** [comment]
We need to have book and section since this is just a very small part of a much larger work and we need the citation patterns to match up with the whole work (if we ever add it here).
I have made the other changes.

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |       +2 |
| `metadata_total`   |       +2 |
| `nodes_count`      |       +3 |
| `texts_passing`    |       +1 |
| `texts_total`      |       +1 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `grc`     |     +130 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0542/__cts__.xml`                          | New      |
| `./data/tlg0542/tlg001/__cts__.xml`                   | New      |
| `./data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg0542/__cts__.xml`                          | Passing  |
| `./data/tlg0542/tlg001/__cts__.xml`                   | Passing  |
| `./data/tlg0542/tlg001/tlg0542.tlg001.1st1K-grc1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3782)*

**Pull request merged**